lib/classic.gi

```diff
@@ -0,0 +1,993 @@
+#############################################################################
+##
+##  classic.gi            'Forms' package
+##
+##  Provide the methods for 'GO', 'SO', 'Omega', 'GU', 'SU', and 'Sp' that
+##  involve a prescribed invariant form.
+##
+
+
+#############################################################################
+##
+#F  PowerTransposedMat( <mat>, <q> )
+##
+##  Return the matrix which has at position (i,j) the entry <mat>[j,i]^<q>.
+##
+BindGlobal( "PowerTransposedMat", function( mat, q )
```

Comment:
This is transpose plus field involution, which I'd call a Chevalley involution and which other people call "adjoint", and yet other "conjugate transpose".

Reply:
@fingolfin I just need the functionality in two places, and I do not care much about the name.

In fact, I would prefer if the GAP library would provide such a function; it could be used for example in the membership test function for the groups GU and SU (see `lib/grpmat.gi` of GAP).

Thinking more about it, perhaps I really prefer a function name that expresses what the function does, and does *not* suggest a mathematical meaning.
(If the function would be documented for the case of a (square?) finite field matrix and an exponent that is a power of the characteristic then in the end one would have to add some tests inside the function whether these conditions are satisfied.)

Reply:
Note that `forms` also has `TransposedFrobeniusMat` which does the exact same thing (just with different code)

Reply:
Hehe, and *also* `Forms_HERM_CONJ` does it